MAINTAINERS.md

```diff
@@ -211,108 +199,164 @@ Examples of a breaking changes might be:
 
 There will be jerks.
 
 #### Rude or Entitled People
 
-These are users who feel the Mocha project and its maintainers _owe them_ time or support. This is incorrect.
+These are users who feel the Mocha project and its maintainers _owe them_ time or support.
+This is incorrect.
 
-However, this behavior is often indicative of someone who is "new" to open source. Many just don't know better. It is not your _responsibility_ to educate them (again, you owe them nothing).
+However, this behavior is often indicative of someone who is "new" to open source.
+Many just don't know better.
+It is not your _responsibility_ to educate them (again, you owe them nothing).
 
 Here are some suggestions:
 
 1. If u mad, wait 20 minutes before writing a comment.
-1. "Kill them with kindness". Explain how they are presenting themselves; maybe link to a good article or two about it.
-1. Don't make it about "users vs. maintainers". Treat them like a potential future maintainer.
-1. Avoid adding to the drama. You could try to reach out privately; email may be in their GitHub profile. You will likely never hear from that individual again (problem solved)
+1. "Kill them with kindness".
+  Explain how they are presenting themselves; maybe link to a good article or two about it.
+1. Don't make it about "users vs.
+  maintainers".
+  Treat them like a potential future maintainer.
+1. Avoid adding to the drama.
+  You could try to reach out privately; email may be in their GitHub profile.
+    You will likely never hear from that individual again (problem solved)
 1. If an issue is getting out of control, lock it.
-1. If someone is _repeatedly_ rude and does not correct their mistakes, you may ban them from participating in the `mochajs` org. If you do not have permission to do so, contact one which does (an "owner").
+1. If someone is _repeatedly_ rude and does not correct their mistakes, you may ban them from participating in the `mochajs` org.
+  If you do not have permission to do so, contact one which does (an "owner").
 
 #### Code of Conduct Violations
 
 **This section is theoretical, as it's yet to happen**.
 
 1. Inform the individual of the violation; link to the CoC
-1. Follow up with JS Foundation for further guidance
+1. Follow up with OpenJS Foundation for further guidance
 1. Repeated violators will be banned inasmuch as that is technically possible
 1. No maintainer nor contributor is exempt from the CoC
 
 ## Branches
 
-`master` is the only maintained branch in `mochajs/mocha` or any of the other repos. **`master` is the only branch to which force-pushing is disallowed.**
+`master` is the only maintained branch in `mochajs/mocha` or any of the other repos.
+**`master` is the only branch to which force-pushing is disallowed.**
 
 Maintainers may push new branches to a repo, as long as they remove them when finished (merging a PR will prompt to do so).
 
 Please _please_ **_please_** delete old or unused branches.
 
 ## Merging PRs
```

Comment:
We're discussing this now. Our consensus is:

1. We don't believe the benefits of a pristine Git history to be worth the cost of asking everyone to maintain that history
2. Squash merging PRs using their title + description preserves a relatively clean history
3. We'll switch to squash merging always, and only enforcing conventions on the PR title
4. We can therefore remove commit linting for anything other than file formatting